### PR TITLE
DOCS-10044 Anypoint Security Jan 20 Release

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 1]
```diff
@@ -166,14 +166,12 @@ Add a list of certificates that were revoked by certificate authorities before t
 
 == Edit a Secret Group
 
-When you edit a secret group, other users are locked out while you edit the necessary security objects. If a user doesn't end an editing session by either finishing or canceling it, the *Edit* button is disabled for other users. If you click *Cancel Edit*, the open editing session is closed, and the *Edit* button becomes available, even if the session was started by a different user. 
+Edit a secret group to add secret types such as Keystore, Truststore, and Certificates.
 
+. In the *Secret Groups* list view, select the secret group you want to edit, and click *Edit*. 
+. Make your changes as required, then click *Finish.*
```
Comment (aishwarya-murali) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Can we remove this line? 'Make your changes as required, then click *Finish.*'. The finish button will not exist after the release. Intead, can we add this line - 'Make your changes as required, then click *Save*, to save each secret type.'

**Review by aishwarya-murali — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/asm-secret-group-concept.adoc [hunk 2]
```diff
@@ -1,34 +1,14 @@
...
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
 
 A secret group is a unique secrets vault to which all your secrets are bound. Secrets Manager stores your secrets per secret group, per environment, and per business group.
 
 User access and authorization controls are applied at the environment level. A user who has access on a given environment can access all the secret groups that belong to that environment. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
 
-Secrets Manager provides both high availability and consistency of stored data, as well as concurrent use functionality for secret renewals. This ensures that all clients use a consistent set of secrets and all users edit the same set of secrets at a given time. 
+Secrets Manager provides both high availability and consistency of stored data, as well as concurrent use functionality for secret renewals. This ensures that all clients use a consistent set of secrets and all users edit the same set of secrets at a given time.
```
Comment (clies212) [inline modules/ROOT/pages/asm-secret-group-concept.adoc]: This paragraph should be deleted. Concurrent use and consistency were eliminated with secret group locks.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 3]
```diff
@@ -1,11 +1,11 @@
-= Create a Secret Group (Anypoint Platform)
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Create and Edit Secret Group (Anypoint Platform)
 
+Using Secrets Manager you can create a secrets group and add secret types to it.
```
Comment (clies212) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Using Secrets Manager you can create a secrets group, a logical grouping of secrets, and add secret types to it.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 4]
```diff
@@ -166,14 +166,12 @@ Add a list of certificates that were revoked by certificate authorities before t
...
 
+. In the *Secret Groups* list view, select the secret group you want to edit, and click *Edit*. 
+. Make your changes as required, then click *Finish.*
 
-. In the *Secret Group* list view, click *Edit* for the secret group that you want to update. 
-. Make your changes and click *Finish*. +
-The secret group is updated with the changes, and the secrets manager serves the updated secrets to all redeployed applications after this point. 
-. Click *Cancel Edit* to close an open editing session. +
-Use caution when you cancel another user's editing session, because all the changes made by the other user will be lost.
+The secret group is updated with the changes, and the secrets manager provides the updated secrets to all redeployed applications after this point.
```
Comment (clies212) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: This sentence is not correct anymore. The save secrets for each type needs an update like that seen below: 

"When each secret is saved, the updated secret is available for use by the next application deployment or the next access of the secret."

**Review by clies212 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/asm-secret-group-concept.adoc (hunk 2 above)
Comment (hping2) [reply modules/ROOT/pages/asm-secret-group-concept.adoc]: agreed. We should delete this paragraph

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 5]
```diff
@@ -166,14 +166,12 @@ Add a list of certificates that were revoked by certificate authorities before t
 
 == Edit a Secret Group
 
-When you edit a secret group, other users are locked out while you edit the necessary security objects. If a user doesn't end an editing session by either finishing or canceling it, the *Edit* button is disabled for other users. If you click *Cancel Edit*, the open editing session is closed, and the *Edit* button becomes available, even if the session was started by a different user. 
+Edit a secret group to add secret types such as Keystore, Truststore, and Certificates.
```
Comment (hping2) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: can we change from "to add" to "to manage", also after Certificates, please add " etc"?
Edit a secret group to manage secret types such as Keystore, Truststore, Certificates and etc.

**Review by aishwarya-murali — DISMISSED** [review]

File: modules/ROOT/pages/asm-secret-group-concept.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/asm-secret-group-concept.adoc]: Removed this paragraph.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 5 above)
Comment (sleslie23) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Changed.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 4 above)
Comment (sleslie23) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Changed to:

After clicking *Finish*, Secrets Manager saves your changes. The updated secret group is immediately available the next time the secret group is accessed or when deploying an application.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Rewrote as:

Secrets Manager enables you to create and edit secrets groups, which is a logical grouping of secrets. Secrets Manager also enables you to create new secret types and add them to a secrets group.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Revised as:

. Make changes to the secrets as required.

As you create or save changes to a secret, the changes are applied immediately. The updated secret group is immediately available the next time the secret group is accessed or when deploying an application.

**Review by hping2 — DISMISSED** [review]
LGTM

**Review by aishwarya-murali — DISMISSED** [review]

**Review by clies212 — DISMISSED** [review]

File: modules/ROOT/pages/asm-secret-group-concept.adoc [hunk 6]
```diff
@@ -1,34 +1,11 @@
 = Secret Groups
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
 
-A secret group is a unique secrets vault to which all your secrets are bound. Secrets Manager stores your secrets per secret group, per environment, and per business group.
+A secret group is a logical grouping of secrets that enable you to manage a groups of secrets as a unit. Secrets Manager stores your secrets per secret group, per environment, and per business group.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/asm-secret-group-concept.adoc]:
```suggestion
A _secret group_ is a logical grouping of secrets that enables you to manage a group of secrets as a unit. Secrets Manager stores your secrets per secret group, per environment, and per business group.
```

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 7]
```diff
@@ -1,27 +1,27 @@
-= Create a Secret Group (Anypoint Platform)
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Create and Edit Secret Group (Anypoint Platform)
 
+Secrets Manager enables you to create and edit secrets groups, which is a logical grouping of secrets. Secrets Manager also enables you to create new secret types and add them to a secrets group.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc:3]: Out of scope but something to think about: could create a partial here so you're not having to restate what a secret group is.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 8]
```diff
@@ -1,27 +1,27 @@
...
-. Type a name for the secret group and click *Save*.
+. Enter a name for the secret group and click *Save*.
 +
 The name of your secret group must: 
 +
-* Be at least three characters long and no longer than 35 characters
 * Start with a letter
-* Contain only letters, numbers, and dashes (but cannot end with a dash)
+* Be at least three characters long and no longer than 35 characters
+* Contain only letters, numbers, and dashes, however, the name cannot end with a dash.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]:
```suggestion
* Contain only letters, numbers, and dashes; however, the name cannot end with a dash.
```

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 9]
```diff
@@ -1,27 +1,27 @@
...
 
-After creating your secret group, it appears in the *Secret Groups* list view. You can then add the necessary secret types by adding a truststore or a keystore.
+After creating your secret group, it appears in the *Secret Groups* list view. Edit the secret group to add the necessary secret types, for example adding a truststore or a keystore.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]:
```suggestion
After creating your secret group, it appears in the *Secret Groups* list view. Edit the secret group to add the necessary secret types, such as a truststore or a keystore.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Take this comment with a grain of salt; "adding" seemed redundant, but I don't have the context to know that for sure.

**Review by sarathecoubian — DISMISSED** [review]
Approving with comments.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**